### PR TITLE
Send email inviting to recruit more volunteers

### PR DESCRIPTION
**Pull request by blackjid**
- The email is set to be sent the next morning at 9:30
- The email use the default template and for now it only shows the link

### CODE REVIEW

**Comment by blackjid** [comment]
ya tengo los tests tranquilein con el merge... ya los subo

File: spec/controllers/volunteer_controller_spec.rb [hunk 1]
```diff
@@ -34,10 +34,35 @@
...
-        expect { post(:create, params: params) }.to have_enqueued_job.on_queue('mailers')
+
+        post(:create, params: params)
+      end
+
+      it 'sends invite to recruit email' do
+        email_link = new_recruiter_url({
+          email: Base64.encode64(params[:volunteer][:email]),
+          host: request.host
+        })
```
Comment (monkeyci) [inline spec/controllers/volunteer_controller_spec.rb]: Indent the right brace the same as the first position after the preceding left parenthesis.

File: spec/controllers/volunteer_controller_spec.rb [hunk 2]
```diff
@@ -34,10 +34,35 @@
...
         expect(VolunteerMailer).to receive(:welcome_email)
           .with(kind_of(Volunteer)).and_call_original
-        expect { post(:create, params: params) }.to have_enqueued_job.on_queue('mailers')
+
+        post(:create, params: params)
+      end
+
+      it 'sends invite to recruit email' do
+        email_link = new_recruiter_url({
+          email: Base64.encode64(params[:volunteer][:email]),
```
Comment (monkeyci) [inline spec/controllers/volunteer_controller_spec.rb]: Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.

File: spec/controllers/volunteer_controller_spec.rb [hunk 3]
```diff
@@ -34,10 +34,35 @@
...
+      it 'sends welcome email' do
         expect(VolunteerMailer).to receive(:welcome_email)
           .with(kind_of(Volunteer)).and_call_original
-        expect { post(:create, params: params) }.to have_enqueued_job.on_queue('mailers')
+
+        post(:create, params: params)
+      end
+
+      it 'sends invite to recruit email' do
+        email_link = new_recruiter_url({
```
Comment (monkeyci) [inline spec/controllers/volunteer_controller_spec.rb]: Redundant curly braces around a hash parameter.

**Comment by blackjid** [comment]
ahora !

**Comment by blackjid** [comment]
[Cuando un voluntario se registra, debe crearse un job que envía un Email automático al día siguiente.](https://trello.com/c/hQIaoSw9/87-cuando-un-voluntario-se-registra-debe-crearse-un-job-que-envia-un-email-automatico-al-dia-siguiente)

**Pull request merged**